### ISSUE TITLE
Fixed Sized Data, exercise 7

### ISSUE THREAD

**Issue by nbl-natsoft** [ayaanjaved07/htdp-excercises#3]
You have misunderstood the question.
Let me rephrase it as such:
Define a function `can-i-go-today` that takes 2 arguments `sunny` and `friday`,
the function returns #true if either 'sunny' is false or 'friday' is true, otherwise the function returns #false
Note: 'sunny' and 'friday' are booleans

**Comment by nbl-natsoft** [comment]
Also fix exercise15 (it actually asks to make the function I describe above), name the function '==>' (it's a valid name in Racket)

**Comment by ayaanjaved07** [comment]
> You have misunderstood the question. Let me rephrase it as such: Define a function `can-i-go-today` that takes 2 arguments `sunny` and `friday`, the function returns #true if either 'sunny' is false or 'friday' is true, otherwise the function returns #false Note: 'sunny' and 'friday' are booleans

question says to define an expression, can define function but not told to do that

**Comment by ayaanjaved07** [comment]
![image](https://github.com/ayaanjaved07/htdp-excercises/assets/74637033/e88aeba4-b5fb-42b8-9d47-4427fa5757e6)
